ft/ft-ops.cc

```diff
@@ -2908,6 +2939,8 @@ ft_handle_open(FT_HANDLE ft_h, const char *fname_in_env, int is_create, int only
     CACHEFILE cf = NULL;
     FT ft = NULL;
     bool did_create = false;
+    bool was_already_open = false;
```

Comment:
Due to code style convention. We use a special tool for auto-formatting, this was moved up automatically.